components/table/TableRow.js

```diff
@@ -53,6 +54,10 @@ const factory = (Checkbox) => {
       if (onChange) {
         return this.renderInput(key, value);
       } else if (value) {
+        if (this.props.model[key].render) {
+          const disabled = true;
```

Comment:
I'm ok with this but I just have a small code suggestion. I never like to pass boolean arguments because of the semantics and I see that to make it clear you assigned here a variable meaning what that third parameter.

Since now we have kind of named parameters with objects and destructuring, what do you think about passing an object instead?

```js
this.props.model[key].render({ 
  index: this.props.index,
  disabled: true
  key
});
```